### DISCUSSION TITLE
No saved authentication after login in the app (structurizr/onpremises) SAML 2.0

### DISCUSSION THREAD

**Discussion by sdobhal369** [structurizr/onpremises#155] [Q&A]
### Expected

Able to browse workspaces

### Actual

The response contained an InResponseTo attribute [ARQ63fee31-3ade-4995-a59a-cabb358ffc81] but no saved authentication request was found, [invalid_assertion] Invalid assertion [id19153587391500991652732734] for SAML response [id19153587389437951789088259]: No subject confirmation methods were met for assertion with ID 'id19153587391500991652732734']
<img width="1782" alt="Screenshot 2024-12-12 at 12 47 03 PM" src="https://github.com/user-attachments/assets/a3905310-e179-4c92-8cbe-ad9b8331c850" />
<img width="1787" alt="Screenshot 2024-12-12 at 12 47 56 PM" src="https://github.com/user-attachments/assets/1a135d73-8eeb-4327-9d61-af7809a5684f" />

### Steps to reproduce

Log in to the app and browse the workspace

### Version/build information

Docker image 

structurizr/onpremises:2024.12.07

### Severity

Minor

### Priority

I have no budget and there's no rush, please fix this for free

**Comment by pmula-onbe** [comment]
I'm having the same issue. I tried to replicate it and seems that I get this error more often when doing these steps:
- Open a couple instances of structurizr on Chrome incognito (not tabs)
- Sign-in into both and wait some minutes
- Sign-out from both but keep instances opened
- Open a new instance of chrome incognito (not tab) and sign-in
- Error pops up

I have the debugged the SAML request/response and I wasn't able to find any issues when this error pops up
